[2021-03-07 20:06:25,304] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: send_to_kafka_1.log 2021-03-07T20:06:23.907283+00:00 [queued]>
[2021-03-07 20:06:25,314] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: send_to_kafka_1.log 2021-03-07T20:06:23.907283+00:00 [queued]>
[2021-03-07 20:06:25,314] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-07 20:06:25,314] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-07 20:06:25,314] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-07 20:06:25,321] {taskinstance.py:1063} INFO - Executing <Task(_PythonDecoratedOperator): log> on 2021-03-07T20:06:23.907283+00:00
[2021-03-07 20:06:25,324] {standard_task_runner.py:52} INFO - Started process 51 to run task
[2021-03-07 20:06:25,326] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'send_to_kafka_1', 'log', '2021-03-07T20:06:23.907283+00:00', '--job-id', '186', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/send_to_kafka_1/main.py', '--cfg-path', '/tmp/tmphdn8maj_', '--error-file', '/tmp/tmpyhow3vae']
[2021-03-07 20:06:25,327] {standard_task_runner.py:77} INFO - Job 186: Subtask log
[2021-03-07 20:06:25,354] {logging_mixin.py:104} INFO - Running <TaskInstance: send_to_kafka_1.log 2021-03-07T20:06:23.907283+00:00 [running]> on host 7531d0e32fd2
[2021-03-07 20:06:25,391] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=iotoi
AIRFLOW_CTX_DAG_ID=send_to_kafka_1
AIRFLOW_CTX_TASK_ID=log
AIRFLOW_CTX_EXECUTION_DATE=2021-03-07T20:06:23.907283+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-07T20:06:23.907283+00:00
[2021-03-07 20:06:25,391] {logging_mixin.py:104} INFO - Should be send in kafka
[2021-03-07 20:06:25,391] {logging_mixin.py:104} INFO - None
[2021-03-07 20:06:25,397] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=send_to_kafka_1, task_id=log, execution_date=20210307T200623, start_date=20210307T200625, end_date=20210307T200625
[2021-03-07 20:06:25,413] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-07 20:06:25,459] {local_task_job.py:146} INFO - Task exited with return code 0
